packages/pds-ember/addon/components/pds/definition-list/index.js

```diff
@@ -0,0 +1,14 @@
+import Component from '@glimmer/component'
+
+/**
+ * @class PdsDefinitionList
+ */
+
+/**
+ * When true, forces the list to render a stacked layout.
+ *
+ * @argument stacked
```

Comment:
Is there a default breakpoint this happens at, or is it entirely controlled by the arg?

Reply:
This is an argument that was brought in to allow application of [styles present in HCP](https://github.com/hashicorp/cloud-ui/blob/master/packages/cloud-ui-core/addon/styles/components/definition-list/index.scss#L97-L99).

* Mobile: 
    * the list is always stacked
* Desktop:
    * if browser doesn't support grid layout, it falls back to being stacked
    * if browser does support grid layout, it applies a grid layout unless overridden by `@stacked={{true}}`